utils/auth.js

```diff
@@ -0,0 +1,51 @@
+const jwt = require('jsonwebtoken');
```

Comment:
jwt isn't going to be used because we aren't storing any info on the client. please alert me before you want to use a new library so next time work doesn't go to waste.